### PR TITLE
adding include to html, change header

### PR DESCRIPTION
**Pull request by RachellCalhoun**
created post_base.html, program_base.html for similar pages. Used 'include' in those html files. Changed header per clients request.

### CODE REVIEW

File: content/static/css/content.css [hunk 1]
```diff
@@ -169,12 +171,23 @@ body {
...
-  /*padding: 1em;*/
-  /*padding-top: 4em;*/
   padding-bottom:0;
-  color: blue;
+  color: #fff;
   font-weight: bold;
+  padding-top: 4em;
+}
+
+.no-img-title {
```
Comment (jinpark) [inline content/static/css/content.css:180]: yay dashes!

File: content/templates/content/our_team.html [hunk 2]
```diff
@@ -4,24 +4,29 @@
...
-        <img class="center-block img-responsive ethics-image" src="{{ team.img.url }}">
-    </div>
-    <div class="col-md-5">
-    {% if team.title %}
-        <h1 class="ethics-title center-block text-center">{{ team.title | safe }}</h1>
-    {% endif %}
-    {% if team.text %}
-        <p class="center-block text-center">{{ team.text | safe }}</p>
-    {% endif %}
+    {% if team.img %} <!--  if image 4, 8 col -->
```
Comment (jinpark) [inline content/templates/content/our_team.html]: I assume that we cant replace this with one of the two templates you made?

File: content/templates/base.html [hunk 3]
```diff
@@ -45,6 +45,9 @@ <h4 class= "subtitle"></h4>
                         </li>
                         {% endif %}
                     {% endfor %}
+                        <li>
+                        <a href="/donate" class="text-uppercase blue-background">Donate</a>
```
Comment (jinpark) [inline content/templates/base.html]: one more indent so the `a` tag is "inside" the `li` tags?

**Comment by RachellCalhoun** [comment]
I think I fixed it up @jinpark Can you look and let me know.

**Comment by jinpark** [comment]
can we also make the donate page using `includes` or no?

**Comment by RachellCalhoun** [comment]
@jinpark done :)

**Comment by jinpark** [comment]
lgtm! yay less code!

**Pull request merged**